tests/class/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -616,7 +616,8 @@ class test
         $customer = (object) ['name' => 'Bob'];
         $job = (object) ['customer' => $customer];
 
-        return "The customer for that job, {$job->customer->name} has an error that shows up after the line gets waaaaay toooo long.";
+        return "The customer for that job, {$job->customer
+            ->name} has an error that shows up after the line gets waaaaay toooo long.";
```

Comment:
Should be fixed in other PR, like here https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAnOMCuqoAIAGAKgBZy5iYDOMEAtnKrgGYSMzECGMuAVhAEYAaXABJgIvvwB0FanQZSoHegF9cnSrg54GqVrnZdclYhADumzAActTeG1K4ANgEsoZAOYZNZjn78AngYQIRDO0B5S+ADcIIIgEFYwLtCUyKAcqHpmAAqZCGkoHABuEC4AJnEg-KgcYADWGADKVnVuHsgwqJhw8W6UDDA5tR60HMhMHE4D8TyUAB4AQrUNzcpwADJucBNTMyCtqAOoyNUc-AFO0FVWqG4wAOoV7MgAHAAM8bcQAw+1VqdbnBjsUdvF0ABHTAudDDDijcZISbTXogAa0FydbqoyjtJxwACKmAg8F2KPiMHOT3KLyQACYKbUXK4oB4AMJ0ManKDQMEgKhwQjnQrIgYqFRAA